conans/model/conan_generator.py

```diff
@@ -0,0 +1,51 @@
+from conans.util.files import save
+from conans.errors import ConanException
+
+
+class Generator(object):
+    def __init__(self, deps_build_info, build_info):
+        self._deps_build_info = deps_build_info
+        self._build_info = build_info
+
+    @property
+    def deps_build_info(self):
+        return self._deps_build_info
+
+    @property
+    def build_info(self):
+        return self._build_info
+
+    @property
+    def content(self):
```

Comment:
Typically I don't feel ABC are necessary, and this approach is enough. In this case, it is fine as-is, but I wouldn't oppose to creating an ABC for the Generator class and abstractproperties for these, they may help for generators without tests providing more detailed error info.